purchases/src/main/kotlin/com/revenuecat/purchases/google/PurchaseContext.kt

```diff
@@ -9,4 +9,5 @@ internal class PurchaseContext(
     val presentedOfferingContext: PresentedOfferingContext?,
     val selectedSubscriptionOptionId: String?,
     val replacementMode: GoogleReplacementMode?,
+    val subscriptionOptionIdForProductIDs: Map<String, String>?,
```

Comment:
This allows us to access which subscription options for add-ons were purchased for which product IDs after a purchase has completed, which then lets us send that information to the RC backend and to the developers using the SDK.